tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     dj32: Django~=3.2.0
     dj40: Django~=4.0.0
     dj41: Django~=4.1.0
+    dj42: .[psycopg3]
```

Comment:
This means the tests don't run against psycopg2 anymore. I will patch this up after merging this PR.

Reply:
https://github.com/SectorLabs/django-postgres-extra/pull/209

I just added this. Tests run against psycopg 2.8, 2.9 and 3.0 now with all supported Django versions.

Reply:
Thanks!